recipes/CMIP6_static_grids/meta.yaml

```diff
@@ -0,0 +1,23 @@
+title: "CMIP6 Static Ocean Grids"
+description: "Raw grid files for the native ocean grids"
+pangeo_forge_version: "0.8.2"
+pangeo_notebook_version: "2021.12.02"
+recipes:
+  - id:
```

Comment:
```suggestion
  dict_object: "recipe:recipes"
```

Reply:
@jbusecke, I'll wait for you to merge this, so that you can see the correct syntax for future reference. I'm following the same convention here as used in: https://github.com/pangeo-forge/cmip6-feedstock/blob/cae65009de76735c72368c5d94ff102fe8203058/feedstock/meta.yaml#L6

Reply:
Oh damn, sorry. I was to hastly with this. Thanks for the fix